make-defs

```diff
@@ -62,6 +62,15 @@ def is_eccodes_post_2_23(args):
     return False
 
 
+def has_eccodes_grib2_templates_subdir(args):
+    """
+    Check if eccodes has grib2/templates subdir
+    """
+    system_grib2 = Path("/") / args.datadir / "eccodes" / "definitions" / "grib2"
+    templates_subdir = system_grib2 / "templates"
+    return templates_subdir.exists() and templates_subdir.is_dir()
```

Comment:
`Path.is_dir()` also returns False if the path doesn’t exist or is a broken symlink, so you can skip the test for `exists()`